bosh-stemcell/spec/stemcells/stig_spec.rb

```diff
@@ -136,7 +136,7 @@
       V-80969
       V-75779
       V-75865
-      V-75851
+      V-260477
```

Comment:
Should this new identifier be replacing V-75851, or should they both be listed?

Reply:
Good catch. That got deleted while fixing a merge conflict. Fixed now